providers/aws/appsync.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/appsync"
+)
+
+type AppSyncGenerator struct {
+	AWSService
+}
+
+func (g *AppSyncGenerator) InitResources() error {
+	config, e := g.generateConfig()
+	if e != nil {
+		return e
+	}
+
+	svc := appsync.New(config)
+
+	// TODO:
+	// * Service does not provides a `NewXXXXXXXXXXXPaginator` like other services' "NewListClustersPaginator" right now
+	// * Replace when it being avaialble, using the max under constraint currently
+	const MaxGraphqlApis = 25
+
+	apis, err := svc.ListGraphqlApisRequest(&appsync.ListGraphqlApisInput{MaxResults: aws.Int64(MaxGraphqlApis)}).Send(context.Background()) // &appsync.).Send(context.Background())
+	if err != nil {
+		log.Println(err)
+		return err
+	}
+
+	var resources []terraform_utils.Resource
+	for _, api := range apis.GraphqlApis {
+		var id = *api.ApiId
+		var name = *api.Name
+		log.Printf(`aws importing appsync_grapql_api: %v`, name)
```

Comment:
Resource import logging is already handled by the tool

Reply:
cleaned up.  referencing some other files, for errors still printout the messages